data/txHandler.go

```diff
@@ -111,6 +112,14 @@ func (handler *TxHandler) Stop() {
 	handler.backlogWg.Wait()
 }
 
+func reencode(stxns []transactions.SignedTxn) []byte {
+	var result [][]byte
+	for stxn := range stxns {
```

Comment:
I think `for stxn := range` needs to be be `for _, stxn := range` or it'll just give you the index

Reply:
oops, embarrassing! thanks adam